sp-rest-api-tutorial.txt

```diff
@@ -2563,6 +2563,41 @@ listings package.
 ** Example: Moving Routers between Collectors                      :noexport:
    - this doesn't work at all, so we shouldn't do it
 
+** Example: Triggering Smart Alerts with Smart Alert Settings
+  #+INDEX: /smart\_alert\_settings/ endpoint
+  #+INDEX: /alerts/ endpoint
+  The SP REST API can be used to create, edit, and delete smart alert settings
```

Comment:
I'd like to see a paragraph or two about what a smart alert is.   something like:

> The smart alert feature uses data from SP Insight to trigger SP alerts that aren't associated with a managed object; there is no associated SHDS, baselines, whatever whatever whatever.  This is only available with SP Insight. 

Or some sort of general introduction to smart alerts - you can probably copy this from whatever the SP documentation on this is.